### DISCUSSION TITLE
Dịch et al.

### DISCUSSION THREAD

**Discussion by lkhphuc** [mlbvn/handson-ml2-vn#1] [Terminology] [answered]
Mình mở thread này thảo luận về việc dịch `et al.`.
Mình đề xuất nên dịch từ này là `và cộng sự`, ví dụ như
"Le-Khac et al. published ..." -> "Lê Khắc và cộng sự đã công bố ..."

Một phương án dịch khác là `và các đồng tác giả` chuẩn hơn nhưng mình thấy hơi dài dòng.

**Comment by hnguyentt** [comment]
Mình cũng đồng ý với Phúc.

**Reply by quangvnai** [reply]
Yes, hoàn toàn đồng ý. `et al.` các thầy hay dịch là `và cộng sự` khi viết báo khoa học.

**Comment by thanhcsf** [comment, marked as answer]
Mình cũng thích từ `cộng sự`.
